issue(workspace): error validating renovate config while running nx g @nx-squeezer/workspace:renovate

Good Morning @samuelfernandez,

I tried the new Version after your very fast fix for #252 👌, but got a new Error. 

Version: "@nx-squeezer/workspace": "^1.0.2",
Win11 
node: 18

Error log: 
```
nx g @nx-squeezer/workspace:renovate

>  NX  Generating @nx-squeezer/workspace:renovate

√ Do you want to use Nx Cloud? (Y/n) · true
√ Do you want to use local presets? To have latest updates it is preferred to use the ones provided by @nx-squeezer/squeezer instead of local ones. (y/N) · false
√ Assignee for GitHub PRs · WolfSoko
In order to use Nx Cloud add a secret with the NX_CLOUD_AUTH_TOKEN
It is suggested that you create a new token for this purpose with read/write access.
You can configure it at: https://cloud.nx.app/
Also consider using Nx Cloud GitHub app to access run results: https://github.com/apps/nx-cloud

CREATE .github/renovate-config.js
CREATE .github/renovate.json
CREATE .github/workflows/renovate.yml
CREATE SECURITY.md
UPDATE .github/workflows/ci.yml
UPDATE README.md
UPDATE tsconfig.base.json
Validating Renovate configuration...
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/renovate-config-validator - Not found
npm ERR! 404
npm ERR! 404  'renovate-config-validator@latest' is not in this registry.
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url.

npm ERR! A complete log of this run can be found in:
npm ERR!     ..._logs\2023-01-05T08_19_35_013Z-debug-0.log
Error: Command failed: npx.CMD renovate-config-validator
npm ERR! code E404
npm ERR! 404 Not Found - GET https://registry.npmjs.org/renovate-config-validator - Not found
npm ERR! 404
    at fulfilled (\wol-sok-mono\node_modules\tslib\tslib.js:115:62) {
  status: 1,
  signal: null,
  output: [
    null,
    <Buffer >,
    <Buffer 6e 70 6d 20 45 52 52 21 20 63 6f 64 65 20 45 34 30 34 0a 6e 70 6d 20 45 52 52 21 20 34 30 34 20 4e 6f 74 20 46 6f 75 6e 64 20 2d 20 47 45 54 20 68 74 ... 417 more bytes>
  ],
  pid: 53176,
  stdout: <Buffer >,
  stderr: <Buffer 6e 70 6d 20 45 52 52 21 20 63 6f 64 65 20 45 34 30 34 0a 6e 70 6d 20 45 52 52 21 20 34 30 34 20 4e 6f 74 20 46 6f 75 6e 64 20 2d 20 47 45 54 20 68 74 ... 417 more bytes>
}

```

Trying to call https://registry.npmjs.org/renovate-config-validator directly indeed gives back a 404.

--- a/packages/workspace/src/generators/renovate/renovate-config-validator-task.spec.ts
+++ b/packages/workspace/src/generators/renovate/renovate-config-validator-task.spec.ts
@@ -20,7 +20,9 @@ describe('@nx-squeezer/workspace renovateConfigValidatorTask', () => {
 
     expect(renovateConfigValidatorTask(tree)).toBeFalsy();
 
-    expect(exec).toHaveBeenCalledWith('npx', ['renovate-config-validator'], { cwd: '/virtual' });
+    expect(exec).toHaveBeenCalledWith('npx', ['--package', 'renovate', '-c', 'renovate-config-validator'], {
+      cwd: '/virtual',
+    });
   });
 
   it('should return false', () => {

--- a/packages/workspace/src/generators/renovate/renovate-config-validator-task.ts
+++ b/packages/workspace/src/generators/renovate/renovate-config-validator-task.ts
@@ -4,13 +4,16 @@ import { exec } from '../lib';
 
 export function renovateConfigValidatorTask(tree: Tree): boolean {
   console.log(`Validating Renovate configuration...`);
-  const { output, error } = exec('npx', ['renovate-config-validator'], {
+  const { output, error } = exec('npx', ['--package', 'renovate', '-c', 'renovate-config-validator'], {
     cwd: tree.root,
   });
 
   if (error != null) {
+    console.error(error);
     return false;
   }
+
+  console.log(output);
 
   return output.includes(`Config validated successfully`);
 }

